lib/movie_card/movie_card.dart

```diff
@@ -13,75 +13,101 @@ class MovieCard extends StatelessWidget {
 
   @override
   Widget build(context) => GestureDetector(
-      onTap: () {
-        Navigator.of(context).push(
-          MaterialPageRoute(
-            builder: (context) => MovieDetailed(movie: movie),
-          ),
-        );
-      },
-      child: Container(
-        color: const Color.fromRGBO(23, 23, 23, 1),
-        height: 150,
-        child: Padding(
-          padding: const EdgeInsets.all(10),
-          child: Row(
-            children: [
-              Stack(
-                children: [
-                  Image.asset(movie.poster),
-                  const AddToWatchlistTab(),
-                ],
-              ),
-              Padding(
-                padding: const EdgeInsets.fromLTRB(10, 10, 0, 5),
-                child: Column(
-                  mainAxisAlignment: MainAxisAlignment.center,
-                  crossAxisAlignment: CrossAxisAlignment.start,
+        onTap: () {
+          Navigator.of(context).push(
+            MaterialPageRoute(
+              builder: (context) => MovieDetailed(movie: movie),
+            ),
+          );
+        },
+        child: Container(
+          color: const Color.fromRGBO(23, 23, 23, 1),
+          height: 150,
+          child: Padding(
+            padding: const EdgeInsets.all(10),
+            child: Row(
+              children: [
+                Stack(
                   children: [
-                    Padding(
-                      padding: const EdgeInsets.only(bottom: 5),
-                      child: Text(
```

Comment:
Since you are using same fontSizes, try to extract all `Texts` from the file to separate class.
It could be class with parameter `final bool isGray;` or `final Color? color;` with default white color.
Something like that.